README.md

```diff
@@ -1,14 +1,10 @@
 # Azure Service Operator Samples
 
-This project contains example uses of the Azure Service Operator. These examples show how to deploy Azure services alongside your Kubernetes deployments.
+This project contains example uses of the Azure Service Operator. These examples show how to deploy Azure services alongside your Kubernetes deployments. 

```

Comment:
Since we're deleting the examples for ASO v1, maybe we should be explicit that these samples now only apply only to ASO v2?

Reply:
Yep, we have a section below where we explicitly mention ASOv2 under `Examples` heading